pkgs/sage-docbuild/pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "sage-docbuild"
+description = "Sage: Open Source Mathematics Software: Build system of the Sage documentation"
+license = {text = "GNU General Public License (GPL) v2 or later"}
+authors = [{name = "The Sage Developers", email = "sage-support@googlegroups.com"}]
+classifiers = [
+    "Development Status :: 6 - Mature",
+    "Intended Audience :: Education",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+    "Operating System :: POSIX",
+    "Operating System :: MacOS :: MacOS X",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Scientific/Engineering :: Mathematics",
+]
+urls = {Homepage = "https://www.sagemath.org"}
+dependencies = ["sphinx"]
```

Comment:
Doesn't sage_docbuild also depend on jupyter-sphinx?

Reply:
I think we are treating the dependencies on Sphinx extensions (declared in sage_docbuild/conf.py) as belonging to sagemath_doc_html

Reply:
I understand the logic but I am not very comfortable with it. Part of me feels that you are going that route, sphinx should be treated the same way. It is not needed at install either, I think.

Reply:
A difference is that sphinx is imported by `sage_docbuild.sphinxbuild`.

Reply:
I do not think I will argue with that one. Where to declare runtime dependency seems to be a bit unsettled.